3rdparty/nuxeo/nuxeo-platform-elasticsearch/src/main/java/org/collectionspace/services/nuxeo/elasticsearch/DefaultESDocumentWriter.java

```diff
@@ -253,6 +254,42 @@ private void denormObjectNameFields(DocumentModel doc, ObjectNode denormValues)
 		denormValues.putArray("objectNameList").addAll(denormObjectNames);
 	}
 
+	/**
+	 * Denormalize the content concept, content event, content person, content organization, and content description
+	 * fields for a collectionobject so that they are indexed under a single field
+	 *
+	 * @param doc the collectionobject document
+	 * @param denormValues the json node for denormalized fields
+	 */
+	private void denormConceptFields(final DocumentModel doc, final ObjectNode denormValues) {
+		final List<JsonNode> denormContentSubject = new ArrayList<>();
+		final List<String> fields = Arrays.asList("contentConcepts",
+			"contentEvents",
+			"contentPersons",
+			"contentOrganizations");
+
+		for (String field : fields) {
+			List<String> contentList = (List<String>) doc.getProperty("collectionobjects_common", field);
+
+			for (String content  : contentList) {
+				if (content != null) {
+					final ObjectNode node = objectMapper.createObjectNode();
+					node.put("subject", RefNameUtils.getDisplayName(content));
+					denormContentSubject.add(node);
+				}
+			}
+		}
+
+		String description = (String) doc.getProperty("collectionobjects_common", "contentDescription");
```

Comment:
@ray-lee looking back at the discussion about the content fields I'm not sure if we need this. I'm going to message Jessi about the display again to double check. I'll add an update here + in jira once I get a response.